optimum/pipelines/pipelines_base.py

```diff
@@ -366,16 +406,22 @@ def pipeline(
         **kwargs,
     )
 
+    use_fast = kwargs.get(use_fast, "True")
     if tokenizer is None and load_tokenizer:
-        tokenizer = get_preprocessor(model_id)
+        tokenizer = AutoTokenizer.from_pretrained(model_id, use_fast=use_fast, _from_pipeline=task, **hub_kwargs)
```

Comment:
is the `_from_pipeline` necessary here ? asking because it might be dropped from transformers at any point

Reply:
removed in https://github.com/huggingface/optimum/pull/2270/commits/28bba3c3eda7d539130c7d6235f0690d5660565d